db/migrations/20210129201213_single_name_field.sql

```diff
@@ -0,0 +1,15 @@
+-- migrate:up
+-- Add new single name field
+ALTER TABLE users ADD COLUMN name VARCHAR;
+COMMENT ON COLUMN users.name IS 'Name of user, may or may not be full name.';
+
+-- Migrate names over
+UPDATE users SET name = users.first_name || ' ' || users.last_name;
+
+ALTER TABLE users ALTER COLUMN name SET NOT NULL;
+
+-- Drop old name fields
+ALTER TABLE users DROP COLUMN first_name CASCADE, DROP COLUMN last_name CASCADE, DROP COLUMN preferred_name CASCADE;
```

Comment:
I don't think we should drop the `preferred_name` as `name` can be considered their "legal name" which may not be what they want to go by (trans and international students).